apps/condo/domains/resident/schema/Resident.test.js

```diff
@@ -79,6 +80,78 @@ describe('Resident', () => {
             const [objCreated] = await createTestResident(adminClient, userClient.user, userClient.property, attrs)
             expect(objCreated.address).toEqual(userClient.property.addressMeta.value)
         })
+
+        describe('unitType and unitName', () => {
```

Comment:
Looks like much of similar tests within many files. Why don't to move them to the single place (domains/common/utils/testSchema/...)?

Reply:
I don't know how to write tests for hooks without using a schema...
But it is worth noting that not all schemas have the same tests, some have different logic

Reply:
I can pass some fake arguments without using a schema, but that won't closely resemble real use